spec.md

```diff
@@ -57,8 +57,11 @@ The high-level components of the spec include:
 * [Filesystem Layer](layer.md) - a changeset that describes a container's filesystem
 * [Image Configuration](config.md) - a document determining layer ordering and configuration of the image suitable for translation into a [runtime bundle][runtime-spec]
 * [Descriptor](descriptor.md) - a reference that describes the type, metadata and content address of referenced content
-* Signatures that are based on signing image content address (optional OCI layer)
-* Naming that is federated based on DNS and can be delegated (optional OCI layer)
+
+Future versions of this specification may include the following OPTIONAL features:
```

Comment:
@RobDolinMS might be concerned about this lowercase “may” (e.g. #109, #110, #245).  Do folks have other ideas for rephrasing this sentence to avoid “may”?  I don't want to use MAY, because this is about the spec roadmap, not [about conformance with this spec version][1].  Perhaps:

> The following OPTIONAL features are under consideration for future versions of this specification:

?

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc5/spec.md#notational-conventions